src/core/lightning/index.types.d.ts

```diff
@@ -19,3 +19,15 @@ interface IPaymentRequest {
   memo?: string
   isReward?: boolean
 }
+
+type AddInvoiceSelfArgs = {
```

Comment:
We're not adding any new code to `core`. Needs to go under `app/wallets/index.types.d.ts`